### PR TITLE
merged with eCHIS environment `environment/echis` directory

### PR DESCRIPTION
**Pull request by demis-svenska**
##### Announce New Release

- [ ] After merging, I will follow [these instructions](https://confluence.dimagi.com/display/saas/Announcing+changes+affecting+third+parties#Announcingchangesaffectingthirdparties-announcing)
to announce a new commcare-cloud release.

### CODE REVIEW

**Comment by demis-svenska** [comment]
this pull request is intended to sync eCHIS environment

File: hosting_docs/source/installation/3-apply-fsck-onreboot.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
+.. _cchq-manual-install:
+
+applying file system check for every reboot
```
Comment (sravfeyn) [inline hosting_docs/source/installation/3-apply-fsck-onreboot.rst:3]: 'applying' to be capitalized.

File: hosting_docs/source/installation/3-apply-fsck-onreboot.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
+.. _cchq-manual-install:
```
Comment (sravfeyn) [inline hosting_docs/source/installation/3-apply-fsck-onreboot.rst:1]: This indicates that this is a new hyperlink, it's already used for actual install docs, so this should be removed.

File: hosting_docs/source/installation/3-apply-fsck-onreboot.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+applying file system check for every reboot 
+====================================================
+
+Checking the file system for errors is an important part of Linux system administration. It is a good troubleshooting
```
Comment (sravfeyn) [inline hosting_docs/source/installation/3-apply-fsck-onreboot.rst:6]: Can we  move this doc to firefighting section to here https://github.com/dimagi/commcare-cloud/blob/master/hosting_docs/source/reference/firefighting/general.rst#in-case-of-a-reboot

Comment (demis-svenska) [reply hosting_docs/source/installation/3-apply-fsck-onreboot.rst:6]: the context is somehow different from the after-reboot. But we can put in https://github.com/dimagi/commcare-cloud/blob/master/hosting_docs/source/reference/firefighting/general.rst

**Comment by demis-svenska** [comment]
closing this PR and create a new one for better visibility

**Pull request closed without merging**